pkgs/by-name/du/duckdb/package.nix

```diff
@@ -109,8 +109,7 @@ stdenv.mkDerivation (finalAttrs: {
           "test/sql/copy/csv/test_sniff_httpfs.test"
           "test/sql/httpfs/internal_issue_2490.test"
           # fails with incorrect result
-          # Upstream issue https://github.com/duckdb/duckdb/issues/14294
-          "test/sql/copy/file_size_bytes.test"
+          "test/issues/general/test_17757.test"
```

Comment:
This test was added as regression checker for https://github.com/duckdb/duckdb/issues/17757

It failed for me on aarch64-linux. I'll test it on other platforms and report back